nss-day-cohort-52/rare-server-smoky-pythons
# can now do a GET request to the server to get all posts or get a sing…

…le post

### Issue ticket(s): #24 

### New file(s): post.py, post_requests.py

### Modified File(s):__init__.py, request_handler.py

### Modifications: defined `get_all_posts` and `get_single_post` functions, and wrote the logic for the `do_GET` method in `request_handler.py`.

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to al-viewPosts
3. Open postman and use `http://localhost:8088/posts` as the url to fetch from
4. perform a GET request on the url above in Postman, you should get back the posts as a list of objects
